src/components/Todo/Todo.module.scss

```
@@ -0,0 +1,14 @@
+.wrapper {
+  border: 2px solid gray;
+  padding: 15px;
+
+  margin-bottom: 15px;
```

```suggestion
  margin-bottom: 16px;
```
Давай округлять до 
`0 | 2 | 4 | 8 | 12 | 16 | 20 | 24 | 32 | 40 | 48 | 64 | 80`
Обычно даже если в фигме у дизайнера чуть съехали значения, то огругляют всё равно.

---

окей, я просто на дизайн особо внимания не обращал

---

https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62
это из дизайн систем идет. Здесь подробней можешь почитать потом.